### PR TITLE
AppInsights info

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by Annie602942 — COMMENTED** [review]
05T03:41:30.3592718Z [36;1mEOF[0m
2024-06-05T03:41:3

**Comment by Annie602942** [comment]
>

File: README.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ GitHub | `saml_return_to_legacy` | This cookie is set by the SAML auth path meth
 GitHub | `show_cookie_banner` | Set based on the client’s region and used to determine if a cookie consent banner should be shown | Session
 GitHub | `tz` | This cookie allows us to customize timestamps to your time zone. | Session
 GitHub | `user_session` | This cookie is used to log you in. | Two weeks
+[Microsoft](https://privacy.microsoft.com/en-us/privacystatement) | `ai_session` | Application Insights session ID | One year
+[Microsoft](https://privacy.microsoft.com/en-us/privacystatement) | `ai_user` | Application Insights user ID | 30 minutes
```
Comment (Annie602942) [inline README.md:37]: https://github.com/privacy/cookies/commit/47c33e2b7c599212dbfb5659e23c0210d0303b09 https://github.com/privacy/cookies/commit/47c33e2b7c599212dbfb5659e23c0210d0303b09

Comment (Annie602942) [reply README.md:37]: > https://github.com/privacy/cookies/commit/47c33e2b7c599212dbfb5659e23c0210d0303b09 https://github.com/privacy/cookies/commit/47c33e2b7c599212dbfb5659e23c0210d0303b09

**Review by markoenix — COMMENTED** [review]
MarkoDeLaTorre

**Review by markoenix — COMMENTED** [review]
MarkoDeLaTorre

**Comment by PYRTUS1997** [comment]
> MarkoDeLaTorre

47c33e2b7c599212dbfb5659e23c0210d0303b09

**Comment by Mamonaminyar** [comment]
https://github.com/privacy/cookies/commit/47196096f6264acbdc6dd3173fc78f1c5b2ea9c3

**Comment by mohammedfarahani** [comment]
در تاریخ یکشنبه ۸ ژوئن ۲۰۲۵،‏ ۱۲:۲۲ Mamon ***@***.***> نوشت:

**Pull request merged**